backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4723,4 +4723,92 @@ databaseChangeLog:
             tag: drop-test_order_id-not_null_constraint-from-result
         - dropNotNullConstraint:
             tableName: result
-            columnName: test_order_id
\ No newline at end of file
+            columnName: test_order_id
```

Comment:
I would add an additional migration to remove the `errors` column on the upload table.  There shouldn't be any entries in it currently and with this pr it would start to fill up with duplicate data that's going into the new table.